### ISSUE TITLE
Suggestions for duplication of resources and license

### ISSUE THREAD

**Issue by svaksha** [WomenWhoCode/guidelines-resources#45]
Hi,

Just saw your [tweet](https://twitter.com/WomenWhoCode/status/767822507160272896?lang=en) and came here - you have some great resources so I have two suggestions:

### 1. Duplication of resources
The resources listed here are great, but instead of creating duplicate learning resources why not contribute to the existing repos that are already curating and listing resources? The [awesome](https://github.com/sindresorhus/awesome) repo** lists every language, field/industry, etc.. that would be relevant to STEMM so it would be useful to reach a wider audience. Besides, keeping things in a single place makes tracking easier. Most of these repos only list the freely available libs and learning resources, not paid ones, AFAIK.

** __Disclaimer__: That repo lists one of the programming language resources I maintain - [Julia](https://github.com/svaksha/Julia.jl) and [Python](https://github.com/svaksha/pythonidae). I would love to merge PR'S from contributors.

### 2. License
The MIT License (MIT) is a software license, usually never used for non-code repos. Since this repo contains just text, you might want to reconsider using a more appropriate license like CC.

Thanks for reading!
Best Wishes!

**Comment by antislice** [comment]
It looks like most of the resources in awesome.re list of lists are actual tools/frameworks/etc, as opposed to resources to master a particular technology. We're going for more of a http://teachyourselftocode.com vibe. However, awesome.re does seem like a useful place to link to.

Good point about the license, @womenwhocode/hq will have to weigh in on that change.

**Comment by svaksha** [comment]
@antislice, True, most cater to libraries and packages and since each repo is maintained by different authors, each one has their own style so I cant speak for them. However, I list learning resources for Python and Julia so feel free to submit PR's to the `Resources.md` page. 

Re license, non-code repos using a software licenses lose the legal privileges inbuilt in, say, a CC license. IANAL, but an MIT license allows anyone to copy (fork if you wish) this repo, change the repo name, remove all the commit info and present it as their own work. Not saying this will happen, but past [experience](https://github.com/JuliaLang/julialang.github.com/issues/194) makes me wary.

Just my zero paise worth :)
